src/platform_impl/windows/keyboard.rs

```diff
@@ -265,7 +265,11 @@ impl KeyEventBuilder {
                             let num_lock_on = kbd_state[VK_NUMLOCK as usize] & 1 != 0;
                             let vkey = event_info.vkey;
                             let physical_key = &event_info.physical_key;
-                            let key = layout.get_key(mod_no_ctrl, num_lock_on, vkey, physical_key);
+                            let key = match physical_key {
+                                    PhysicalKey::Code(KeyCode::BracketRight) => Key::Character("]".to_smolstr()),
```

Comment:
I'm 99% sure that this is not correct fix for cases where the layout doesn't have direct mapping, for example on dvorak.

All information manipulating with the layouts should come from the win32 API.

Reply:
I tested the whole thing with right-handed Dvorak layout and CTRL+[, CTRL+] work well.

Reply:
it's strange that it works on dvorak, given that physical keys for `[]` are `-=` respectively, I won't expect it to work.

Reply:
OK. I will check it again.

Reply:
It does not work in   `` right-handed Dvorak`` layout. What puzzled me is that physical_key:Code()  in ``Dvorak`` is reported the same as in ``Czech`` layout
czech layout:
```
TRACE [neovide::window::keyboard_manager] KeyEvent {
    physical_key: Code(
        ControlLeft,
    ),
    logical_key: Named(
        Control,
    ),
    text: None,
    location: Left,
    state: Pressed,
    repeat: false,
    platform_specific: KeyEventExtra {
        text_with_all_modifers: None,
        key_without_modifiers: Named(
            Control,
        ),
    },
}
TRACE [neovide::window::keyboard_manager] KeyEvent {
    physical_key: Code(
        BracketRight,
    ),
    logical_key: Character(
        ")",
    ),
    text: Some(
        "]",
    ),
    location: Standard,
    state: Pressed,
    repeat: false,
    platform_specific: KeyEventExtra {
        text_with_all_modifers: Some(
            "\u{1d}",
        ),
        key_without_modifiers: Character(
            ")",
        ),
    },
}
TRACE [neovide::window::keyboard_manager] Key pressed <C-]> ModifiersState(CONTROL)

```
 right-handed Dvorak layout:
```
TRACE [neovide::window::keyboard_manager] KeyEvent {
    physical_key: Code(
        ControlLeft,
    ),
    logical_key: Named(
        Control,
    ),
    text: None,
    location: Left,
    state: Pressed,
    repeat: true,
    platform_specific: KeyEventExtra {
        text_with_all_modifers: None,
        key_without_modifiers: Named(
            Control,
        ),
    },
}
TRACE [neovide::window::keyboard_manager] KeyEvent {
    physical_key: Code(
        BracketRight,
    ),
    logical_key: Character(
        "=",
    ),
    text: None,
    location: Standard,
    state: Pressed,
    repeat: false,
    platform_specific: KeyEventExtra {
        text_with_all_modifers: None,
        key_without_modifiers: Character(
            "=",
        ),
    },
}
TRACE [neovide::window::keyboard_manager] Key pressed <C-=> ModifiersState(CONTROL)

```

Reply:
Well, the physical keys must stay the same because the original layout is qwerty, what changes are logical keys, the keys after the translation.

There are hardware dvorak keyboards, on them you don't change to e.g. dvorak layout, because it'll reverse the thing, since it's already dvorak in hardware.

The point I was making, if it was not clear, that the physical keys stay the same regardless of layout you pick in windows, while what changes are logical keys, thus to properly implement what you want you should work with logical keys and layout information, both should come from win32 APIs.

Like if it was e.g. Linux I could help with that further, but given that it's a windows, I'm not that familiar with it and use only for testing like _it at least runs_.

Reply:
Thanks for clarification!

I founded my fix on the fact that physical keys are the same for all layouts.

In the 1st version of fix I changed logical key but it does not work in any layout.

This pull request contains the second version which changes the Text property of Key Event .It works in Czech German and English layout but not in Dvorak.

I will try to change logical key and text at once.

Reply:
I added  the second part which causes logical_key change. It is needed for functioning in Dvorak layout.

For Czech and English layout the change of KeyEvent Text property, introduced in the first commit of this PR  is sufficient.

Now the PR works well in Czech, English and Dvorak layout.

Czech layout:
```
TRACE [neovide::window::keyboard_manager] KeyEvent {
    physical_key: Code(
        ControlLeft,
    ),
    logical_key: Named(
        Control,
    ),
    text: None,
    location: Left,
    state: Pressed,
    repeat: false,
    platform_specific: KeyEventExtra {
        text_with_all_modifers: None,
        key_without_modifiers: Named(
            Control,
        ),
    },
}
TRACE [neovide::window::keyboard_manager] KeyEvent {
    physical_key: Code(
        BracketLeft,
    ),
    logical_key: Character(
        "[",
    ),
    text: Some(
        "[",
    ),
    location: Standard,
    state: Pressed,
    repeat: false,
    platform_specific: KeyEventExtra {
        text_with_all_modifers: Some(
            "\u{1b}",
        ),
        key_without_modifiers: Character(
            "[",
        ),
    },
}
TRACE [neovide::window::keyboard_manager] Key pressed <C-[> ModifiersState(CONTROL)
TRACE [neovide::channel_utils] UIComand Serial(Keyboard("<C-[>"))
```
Dvorak layout
```
TRACE [neovide::window::keyboard_manager] KeyEvent {
    physical_key: Code(
        ControlLeft,
    ),
    logical_key: Named(
        Control,
    ),
    text: None,
    location: Left,
    state: Pressed,
    repeat: true,
    platform_specific: KeyEventExtra {
        text_with_all_modifers: None,
        key_without_modifiers: Named(
            Control,
        ),
    },
}
TRACE [neovide::window::keyboard_manager] KeyEvent {
    physical_key: Code(
        BracketLeft,
    ),
    logical_key: Character(
        "[",
    ),
    text: None,
    location: Standard,
    state: Pressed,
    repeat: false,
    platform_specific: KeyEventExtra {
        text_with_all_modifers: None,
        key_without_modifiers: Character(
            "[",
        ),
    },
}
TRACE [neovide::window::keyboard_manager] Key pressed <C-[> ModifiersState(CONTROL)

```